Gems/ROS2/Code/Source/Manipulation/JointStatePublisher.h

```diff
@@ -25,14 +26,18 @@ namespace ROS2
     //! A class responsible for publishing the joint positions on ROS2 /joint_states topic.
     //!< @see <a href="https://docs.ros2.org/latest/api/sensor_msgs/msg/JointState.html">jointState message</a>.
     class JointStatePublisher
+        : public ROS2::Utils::PhysicsCallbackHandler
     {
     public:
         JointStatePublisher(const PublisherConfiguration& configuration, const JointStatePublisherContext& context);
+        virtual ~JointStatePublisher() = default;
 
-        //! Update time tick. This will result in state publishing if timing matches frequency.
-        void OnTick(float deltaTime);
+        void Activate();
```

Comment:
Could you document it? It is not a bug but looks like one, or better choose other name since it can be mistaken AZ::Component::Activate.